src/Box.hs

```diff
@@ -247,7 +247,7 @@ initRow startColumn =
 
 spacesInTab :: Int
 spacesInTab =
-  4
+  2
```

Comment:
It's funny that this is here actually – I came here with the following train of thought:

- We'd like to use 2 spaces instead of 4, can I configure it?
- Hrm, can't configure... maybe I can do my own `elm-format` build
- How do I build this using stack then...
- Oh, a PR about stack
- Heh! 2 spaces 😁